### DISCUSSION TITLE
Pouvoir ajouter des amis et interagir avec eux

### DISCUSSION THREAD

**Discussion by Luleil** [Crownicles/Crownicles#1098] [Nouvelles idées]
Premièrement, il faudrait pouvoir créer un système où l'on pourrait avoir des amis ( avec une liste restreinte ). 

Ensuite on pourrait faire différentes actions à ses amis comme par exemple :
- leur dire bonjour
- leur faire un câlin
- se mettre ensemble 
- se marier
- pouvoir leur donner des items
- pouvoir donner de l'argent
Et encore beaucoup d'autres 

Voilà pour quelques idées ^^

**Comment by beaud37210** [comment]
"se mettre ensemble"; "se marier", c'est pas un peu trop?
Sinon je suis d'accord avec toi sur le don d'item et d'argent

**Reply by Luleil** [reply]
Oui mes principales idées étaient surtout les deux dernières. Les autres étaient aux cas où ça intéresserait ^^

**Reply by beaud37210** [reply]
Merci, je préfère 😅

**Reply by Luleil** [reply]
Oui 😂 après qui sais peut être que plus tard quand il sera très évolué ;3

**Reply by beaud37210** [reply]
Évolué, tu es sur?😅

**Reply by Luleil** [reply]
Bah si on rajoute beaucoup de mise à jour peut être qu'à un moment on y rajouterais de nouvelles fonctionnalités ^^ mais je pense que ça serait bien plus tard ^^'

**Reply by beaud37210** [reply]
Je pense aussi😀

**Reply by leuhvraiminoryo** [reply]
Je suis un poil désolé de vous l'apprendre mais draft a déjà exprimé son opposition vis-à-vis de l'idée d'échange d'argent et d'items avec des gens (une des raisons étant que c'est trop facile à exploiter). C'est pour cela, entre autres, que la vente de pets ne donne pas directement l'argent au vendeur, et qu'il ne reçoit qu'une quantité "équivalente" d'xp de guilde. De plus vu comment le bot a évolué jusqu'à maintenant, et en prenant en compte ce qui est prévu pour le futur, les "amis" me semblent avoir peu de chances d'exister en tant que fonctionnalité dans le bot. 

Je précise cependant que j'exprime seulement mes pensées en prenant en compte certains éléments, il se peut tout à fait que draft se dise finalement qu'il veut faire un truc comme ça 🙂

**Reply by Luleil** [reply]
Ah ;^; c'est dommage je trouve, ça donnerait beaucoup plus d'interaction avec les autres joueurs que juste des combats ( où l'on ne gagne rien ) et des échanges de familier. Je trouve que ça pourrait mettre un peu plus d'ambiance ^^

**Reply by beaud37210** [reply]
Si si si, les combats c'est vraiment bien, tu gagnes plein de points quand tu gagnes mais tu en perds quand tu perds. Sauf pour les ff. Mais c ultra pratique quand tu veux gagner le topweek

**Reply by Luleil** [reply]
Awi ? Oki ça va alors ^^

**Reply by beaud37210** [reply]
Genre si ct un beau combat tu gagnes 1500 point facile

**Reply by Luleil** [reply]
Awi oki, parce que lorsque j'allais sur le serveur du draftbot pour voir les combat je voyais des gens qui combattait mais je ne voyais pas ce qu'ils avaient gagné du coup je me disais que ça en valait pas vraiment la peine sauf pour s'amuser avec ses amis :3

**Reply by beaud37210** [reply]
![Screenshot_2022-04-25-17-28-48](https://user-images.githubusercontent.com/97196060/165122122-cee2478d-5522-419d-af7b-eaf342c4f106.png)
C'est marqué dessus

**Reply by beaud37210** [reply]
Points gagnés perdus

**Reply by Luleil** [reply]
Les points perdus / gagnés sont les mêmes pour les deux ?

**Reply by Karstarck** [reply]
Alors, si vous avez des questions sur le bot, il y a un channel spécifique sur le serveur officiel. Il y aura toujours des gens pour vous aider :) (peut être pas au milieu de la nuit non plus)

**Reply by Luleil** [reply]
Vwi ^^'
